lib/openshift/validating_webhook_configuration.rb

```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class ValidatingWebhookConfiguration < ClusterResource
+    RESOURCE = "ValidatingWebhookConfiguration"
```

Comment:
full qualified API name here too,
please post `oc get -o yaml` output of resource in case you find trouble figuring that out.

Reply:
Similar error for full qualified API
For post `oc get -o yaml` I changed code, not sure correct, please check

Reply:
Full YAML file
http://pastebin.test.redhat.com/897377